db/bootstrapUser.sql

```diff
@@ -47,5 +47,14 @@ SET time_zone = "+00:00";
 -- Dumping data for table `users`
 --
 
+-- CREATE USER 'zookeeper'@'localhost' IDENTIFIED BY 'zookeeper';
+
+drop database if exists zkdb;
+-- create database zkdb;
+GRANT ALL PRIVILEGES ON zkdb.* TO 'zookeeper'@'localhost';
+use zkdb;
+
+create table users;
+
```

Comment:
These changes need to be removed.  The DROP, GRANT, and USE are not appropriate for this script.  This script runs against an existing database (can be any database, not just zkdb) and inserts one row.  It assumes both the database and table already exist.  The user specifies the database instance when the script is run.

Reply:
done. suggest that you update the readme with the preferred sequence of steps to start from scratch. i took me quite a while to get everything step and i am fairly familiar with this stack.

Reply:
Thanks Eric.  You are right; the db configuration steps are missing from the instructions.  I am going to add something similar to what I put up on internal for prod.

In the meantime, you missed the DROP and GRANT in this one.  Can you please remove them?